### PR TITLE
extracting radio_button in its own file

### PR DESCRIPTION
**Pull request by remyyounes**
Todo: Move SVG definitions to their own file and provide a getter function for SVGs.
`themed-svg($id, $color)`

### CODE REVIEW

File: packages/core-css/src/components/_radio_button.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    flex: 0 0 30px;
+    background-size: 100% 100%;
+  }
+
+  &__input + &__label:before {
+    background-image: $img_radio_base;
+  }
+
+  &__input:checked + &__label:before {
+    background-image: $img_radio_checked;
```
Comment (mboperator) [inline packages/core-css/src/components/_radio_button.scss]: We should use dashes to maintain consistency with the rest of the variables

File: packages/core-css/src/components/_checkbox.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    display: none;
+  }
+
+  &__input + &__label {
+    display: flex;
+    align-items: center;
+  }
+
+  &__label:before {
+    content: "";
```
Comment (mboperator) [inline packages/core-css/src/components/_checkbox.scss]: What's this for?

Comment (remyyounes) [reply packages/core-css/src/components/_checkbox.scss]: This is for displaying pseudo elements with background-image

File: packages/core-css/src/components/_radio_button.scss (hunk 1 above)
Comment (remyyounes) [reply packages/core-css/src/components/_radio_button.scss]: This is how you originally coded it so i didn't want to change it in this branch. I'm opening a followup PR.

**Pull request merged**